examples/clients/abe/api/swagger.yaml

```diff
@@ -2388,6 +2388,139 @@ definitions:
     properties:
       value:
         type: "string"
+  A bit of everything:
```

Comment:
This appears to be an artifact of swagger-codegen: It is adding a definition for the inline schema in the source Swagger spec, choosing a name based on the "title" of the inlined schema object.

Reply:
I think this should be fine, it shouldn't cause any breakages.